projects/forest_firefighters/controllers/fire/fire.py

```diff
@@ -41,22 +41,23 @@ def __init__(self, node):
         self.smoke = None
         self.fire_count = 0
         self.translation = node.getField('translation').getSFVec3f()
-        self.scale = node.getField('size').getSFFloat() / 10
+        self.scale = node.getField('scale').getSFFloat()
         self.robustness = random.uniform(self.ROBUSTNESS_VARIATION, self.ROBUSTNESS_VARIATION)
+        node.getField('burnt').setSFBool(False)  # Ensure normal tree as initial state
 
     def stopFire(self):
         if self.fire:
             fire_translation_field = self.fire.getField('translation')
             fire_translation = fire_translation_field.getSFVec3f()
             t = [fire_translation[0], fire_translation[1], fire_translation[2]]
-            t[1] = 10000000
+            # t[2] = 1000000

```

Comment:
You should not leave commented out code without any explanation.
Either remove it or add an explanation.